docs/guides/automating-a-process-using-bpmn.md

```diff
@@ -0,0 +1,102 @@
+---
+id: automating-a-process-using-bpmn
+title: Automating a process using BPMN
```

Comment:
In general:
would it not be great to at least have a very very simple API call? The current scope of this guide only aims at manual workflow orchestration, and I feel the API bit is also important.

Reply:
Agreed. Will loop in Niall for review and input on a better sample/example process.